app/controllers/graphql_controller.rb

```diff
@@ -2,6 +2,10 @@ class GraphqlController < ApplicationController
   skip_forgery_protection if: :skip_csrf_protection?
   skip_before_action :redirect_to_primary_domain, if: :introspection?
 
+  rescue_from ActionController::InvalidAuthenticityToken do
+    render json: { errors: [{ message: 'Invalid or expired session, Please try refresh page or signing in again' }] }
```

Comment:
This would work much better if we had a way to also pass a `refresh: true` over to the UI, where it could be used to show a _Refresh Page_ button below the notification text.

However, I looked at the implementation, and it looks like this object is being converted into a simple array of strings and then iterated over using the `Notification` module. So patching it to accept a new option won't be straight-forward.

I've devised a simple fix for this that isn't compatible with i18n. I'll push it after this review.

Reply:
@harigopal -

I feel the `Refresh Page` Button looks a bit bigger in notification text, Shall we just keep `refresh page icon` as a small button instead of text ?

![image](https://user-images.githubusercontent.com/2093818/128665143-751643d6-4287-4ad2-9ad7-5cc2e93aa668.png)

Reply:
Option 1 : (Just decreased the `mt` in button)

![image](https://user-images.githubusercontent.com/2093818/128665596-83af3631-30a3-497d-8e5c-835c3475da54.png)

Option 2 : (With Icon)

![image](https://user-images.githubusercontent.com/2093818/128665921-81a818d1-6f67-42fb-8120-57a330731a5d.png)


Which option would you recommend?

Reply:
@prashanth726 Weird. On my browser, the button was below the text. Hence the `mt-4`, to give it some margin from the text.

Please restore the full text, and try adding the `block` class to the `button` tag so that it shows up _after_ the text of the notification.